src/WooCommerce.php

```diff
@@ -1492,4 +1492,34 @@ private static function getRepricingOptionsDropdownValues(\WC_Product $product)
     return $dropdown_values;
   }
 
+  /**
+   * Adds custom meta to customer.
+   *
+   * @implements woocommerce_checkout_create_order
+   */
+  public static function addCustomMetaForUser($order) {
+    $user_id = $order->get_user_id();
+    $items_counter = $order->get_item_count();
+    $customer_orders_count = get_user_meta($user_id, '_' . Plugin::PREFIX . '_customer_orders_count', TRUE);
+    $customer_order_items_count = get_user_meta($user_id, '_' . Plugin::PREFIX . '_customer_order_items_count', TRUE);
+    if ($customer_orders_count && $customer_order_items_count) {
```

Comment:
The first time you try to run this     
```
$customer_orders_count = get_user_meta($user_id, '_' . Plugin::PREFIX . '_customer_orders_count', TRUE);
$customer_order_items_count = get_user_meta($user_id, '_' . Plugin::PREFIX . '_customer_order_items_count', TRUE);
```

it will return an empty string and thus not moving into the conditional.

This will always evaluate false: 

`if ($customer_orders_count && $customer_order_items_count)`

if they user meta is empty for both fields. 

https://developer.wordpress.org/reference/functions/get_user_meta/#return